stdlib/ballerina-http/src/main/ballerina/http/auth/authz_filter.bal

```diff
@@ -68,25 +68,35 @@ public type AuthzFilter object {
                 authorized = true;
             }
         }
-        return createAuthzResult(authorized);
+        return isAuthzSuccessfull(listener, authorized);
+    }
+
+    public function filterResponse(Response response, FilterContext context) returns boolean {
```

Comment:
If we are not using these two input paras can we get rid of it?

Reply:
It is the signature of a `http:Filter` object's `filterResponse` method and should be as it is for the match to succeed